docs/sources/enterprise/access-control/roles.md

```diff
@@ -44,6 +44,10 @@ Role names must be unique within an organization.
 
 Roles with names prefixed by `fixed:` are fixed roles created by Grafana and cannot be created or modified by users.
 
+### Role display name
+
+A role’s display name is human friendly text that is displayed in the UI. When you create a display name for a role, use up to 190 ASCII-based characters. For fixed roles, the display name removes the prefix and separates any `:` (a colon) with ` ` (a space).
```

Comment:
> For fixed roles, the display name removes the prefix and separates any `:` (a colon) with ` ` (a space)

This is only the fallback behavior right?

Reply:
this is true. We need to write that whatever you name it will come up but the fallback is this.